src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -64,12 +65,10 @@
       final String acceptVersion, @Valid final MultipartFile file) {
     final String expectedAcceptVersion = storeApiVersion;
     if (!StringUtils.equals(acceptVersion, expectedAcceptVersion)) {
-      log.warn(
-          "Expected Accept-Version to be \"{}\" but was \"{}\". Only \"{}\" is currently supported.",
-          expectedAcceptVersion,
-          acceptVersion,
-          expectedAcceptVersion);
-      return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
+      throw new UnsupportedOperationException(
+          String.format(
+              "Expected Accept-Version to be \"%s\" but was \"%s\". Only \"%s\" is currently supported.",
```

Comment:
nano-nit -> Giving the `expectedAcceptVersion` twice seems excessive, `Expected Accept-Version was \"%s\", but only \"%s\" is currently supported.` (and below)